src/commands/ext-configure.ts

```diff
@@ -65,9 +65,9 @@ export default new Command("ext:configure <extensionInstanceId>")
         const plural = immutableParams.length > 1;
         logger.info(`The following param${plural ? "s are" : " is"} immutable:`);
         for (const { param } of immutableParams) {
-          logger.info(
-            `param: ${param}, value: ${_.get(existingInstance, `config.params.${param}`)}`
-          );
+          const value = _.get(existingInstance, `config.params.${param}`);
+          logger.info(`param: ${param}, value: ${value}`);
+          params[param] = value;
```

Comment:
Was not sending the immutable params on purpose? Or just an oversight?

Reply:
This was a misunderstanding of how updateMasks work - my original intent was to not change those params by not sending them. However, since the updateMask is config.params, this was replacing the entire params object with what we sent, and getting rid of the immutable params. We never noticed because location is the only immutable param at the moment, and in testing, we've mostly used the default location us-central1